python_modules/dagster/dagster/utils/log.py

```diff
@@ -222,8 +223,12 @@ def default_format_string():
     return "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
 
 
+def default_date_format_string():
+    return "%Y-%m-%d %H:%M:%S%z"
```

Comment:
I think something like a space, parens or some delimiting for the `%z` would probably alleviate my minor visual parsing difficulty. 

Don't feel that strongly though so follow your heart